integrations/gitlab/gitlab_integration/events/event_handler.py

```diff
@@ -42,10 +65,10 @@ def on(self, hook_handler: Type[HookHandler]) -> None:
     def add_client(self, client: GitlabService) -> None:
         self._clients.append(client)
 
-    async def notify(self, event: str, body: dict[str, Any]) -> Awaitable[Any]:
+    async def _notify(self, event: str, body: dict[str, Any]) -> None:
         # best effort to notify using all clients, as we don't know which one of the clients have the permission to
         # access the project
-        return asyncio.gather(
+        asyncio.gather(
```

Comment:
why dont you await?

Reply:
check if it needed

Reply:
not needed

Reply:
the difference is that before your returned the task back to be awaited by someone else
please await here